jekyll/_cci2/contexts.md

```diff
@@ -7,104 +7,124 @@ categories: [configuring-jobs]
+CircleCI enables you to restrict secret environment variables at run time by adding security groups to contexts. Only organization administrators may add *security groups* to a new or existing context. Security groups are defined by GitHub teams. If you are using CircleCI Server with LDAP authentication, then LDAP groups also define security groups. After a security group is added to a context, only members of that security group who are also CircleCI users may access the context and use the associated environment variables. 
 
-Note: Organization administrators have read/write access to all projects and have unrestricted access to all contexts.
+Organization administrators have read/write access to all projects and have unrestricted access to all contexts.
 
 The default security group is `All members` and enables any member of the organization who uses CircleCI to use the context.
 
-## Running Workflows with a Restricted Context
+**Note:** Bitbucket repositories do **not** provide an API that allows CircleCI contexts to be restricted, only GitHub projects include the ability to restrict contexts with security groups.
 
-To invoke a job that uses a restricted context, a user must be a member of one of the security groups for the context or the workflow will fail with the status of `Unauthorized`. If you add a context to your job and you are **not** a member of any of the security groups, the workflow will fail as `Unauthorized`.
+## Running Workflows with a Restricted Context
 
-**Note:** Bitbucket repositories do **not** provide an API that allows CircleCI contexts to be restricted, only GitHub projects include the ability to restrict contexts with security groups. Restricted Contexts are also **not** yet supported in private installations of CircleCI.
+To invoke a job that uses a restricted context, a user must be a member of one of the security groups for the context. If the user running the workflow does not have access to the context, the workflow will fail with the `Unauthorized` status.
 
 ### Restrict a Context to a Security Group or Groups
+{:.no_toc}
 
 You must be an organization administrator to complete the following task.
 
 1. Navigate to Organization Settings > Contexts page in the CircleCI app. The list of contexts appears. The default security group is `All members` and allows all users in the org to invoke jobs with that context.
-2. Click the Create Context button if you will use a new context or click the name of an existing context.
-3. Click the Add Group link. The Add Groups dialog box appears.
+2. Click the Create Context button if you wish to use a new context, or click the name of an existing context.
+3. Click the Add Secutrity Group button to view the dialog bo.
```

Comment:
```suggestion
3. Click the Add Security Group button to view the dialog box.
```

Reply:
wowzers you can tell my head is in two places at once, thanks Thomas!